src/e2e/puppeteer/__tests__/color.ts

```diff
@@ -31,10 +33,12 @@ it('Set the text color of the text', async () => {
   await click('[data-testid="toolbar-icon"][aria-label="Text Color"]')
   await click('[aria-label="text color swatches"] [aria-label="blue"]')
 
+  await press('ArrowUp')
+  await press('ArrowDown')
```

Comment:
It should not be necessary to blur and refocus. If so, that probably indicates a bug. Can you explain?